test/parallel/test-https-client-get-url.js

```diff
@@ -35,8 +36,8 @@ const url = require('url');
 const URL = url.URL;
 
 const options = {
-  key: fs.readFileSync(`${common.fixturesDir}/keys/agent1-key.pem`),
-  cert: fs.readFileSync(`${common.fixturesDir}/keys/agent1-cert.pem`)
+  key: fs.readFileSync(`${fixtures.fixturesDir}/keys/agent1-key.pem`),
```

Comment:
This could be using method `fixtures.readKey` instead of `fs.readFileSync` to get the keys.

Reply:
D'oh! Updated to use fixtures.readKey and fixtures.readSync (for the cert).